src/applications/vaos/appointment-list/components/ScheduleNewAppointment.jsx

```diff
@@ -8,14 +8,26 @@ import {
   selectFeatureRequests,
   selectFeatureStatusImprovement,
   selectFeaturePrintList,
+  selectFeatureStartSchedulingLink,
+  selectFeatureBreadcrumbUrlUpdate,
 } from '../../redux/selectors';
 // eslint-disable-next-line import/no-restricted-paths
 import getNewAppointmentFlow from '../../new-appointment/newAppointmentFlow';
 
-function handleClick(history, dispatch, typeOfCare) {
-  return () => {
+function handleClick(
+  history,
+  dispatch,
+  typeOfCare,
+  featureBreadcrumbUrlUpdate = false,
+) {
+  return e => {
+    // Stop default behavior for anchor tag since we are using React routing.
+    e.preventDefault();
+
     recordEvent({
-      event: `${GA_PREFIX}-schedule-appointment-button-clicked`,
+      event: featureBreadcrumbUrlUpdate
+        ? `${GA_PREFIX}-vaos-start-scheduling-link`
```

Comment:
It is recording the event as `vaos-vaos-start-scheduling-link`. Suggest removing "-vaos"
```suggestion
        ? `${GA_PREFIX}-start-scheduling-link`
```